### ISSUE TITLE
dPub Reccommendations

### ISSUE THREAD

**Issue by WayneEDick** [w3c/wcag#163]
<p><strong>Recommendation:</strong> WCAG WG should submit a survey puttting all of these suggestions to an up or down vote with suggestions.</p>
<h1>dPub Suggestions:<a href="http://w3c.github.io/dpub-accessibility/#navigating-among-multiple-documents">[dPub]</a></h1>
<h2>2.1: Package Issues</h2>
<p>Suggestion 1: Add discussion of packaged documents to WCAG. </p>
<p>Suggestion 2: Add author extension to WCAG for packaged documents. Note: Although package navigation seems to be covered by conformance it may be good to add a technique addressing navigation between package elements.</p>
<h2>2.2 Skipability</h2>
<p>Suggestion (UAAG): User agents need to implement skippability, and it cannot be confined to users of AT. </p>
<h2>2.3 Escapeability</h2>
<p>Suggestion (UAAG): User agents need to implement escape to the next sibling or parent element, and it cannot be confined to users of AT.</p>
<h2>2.4 Page Numbers / Pagination</h2>
<p>Suggestion 1 (WCAG): Implement a concept such as EPUB's correspondence with print pages</p>
<p>page-list</p>
<p>and </p>
<p>pagebreak</p>
<p>Suggestion 2 (UAAG): User agents need to enable, where relevant, both distinct concepts of pages (described below), including allowing users to jump to page numbers in either model, or view specific pages in either model.</p>
<h2>2.5 Phonetic spellings of proper nouns and jargon terms</h2>
<p>Suggestion: New or modified specification allowing simple pronunciation schemes to be shared among multiple publications.</p>
<h2>2.6 Deeply Nested Heading Levels Best Practices</h2>
<p>Suggestion: Add to WCAG Techniques and to the WAI Tutorials.</p>
<h2>2.7 Semantic List-Head</h2>
<p>Suggestion 1 (ARIA): Outreach to ARIA WG about proposing aria-labelledby be exposed by user agents directly without AT.</p>
<p>Suggestion 2: Outreach to various groups about alternative non-ARIA semantically meaningful list heading.</p>
<h2>2.8 Drop Cap</h2>
<p>Suggestion 1 (ATAG): Possible addition to ATAG techniques.</p>
<p>Suggestion 2 (WAI Outreach): Outreach to content creators that CSS 3 solves the drop cap problem accessibly.</p>
